registry/storage/paths.go

```diff
@@ -23,25 +23,33 @@ const (
 //
 // The path layout in the storage backend is roughly as follows:
 //
-//		<root>/v2
-//			-> repositories/
-// 				-><name>/
-// 					-> _manifests/
-// 						revisions
-//							-> <manifest digest path>
-//								-> link
-// 						tags/<tag>
-//							-> current/link
-// 							-> index
-//								-> <algorithm>/<hex digest>/link
-// 					-> _layers/
-// 						<layer links to blob store>
-// 					-> _uploads/<id>
-// 						data
-// 						startedat
-// 						hashstates/<algorithm>/<offset>
-//			-> blob/<algorithm>
-//				<split directory content addressable storage>
+//	<root>/v2
+//	├── blob
+//	│   └── <algorithm>
+//	│       └── <split directory content addressable storage>
+//	└── repositories
+//	    └── <name>
+//	        ├── _layers
+//	        │   └── <layer links to blob store>
+//	        ├── _manifests
+//	        │   ├── revisions
+//	        │   │   └── <manifest digest path>
+//	        │   │       └── link
+//	        │   └── tags
+//	        │       └── <tag>
+//	        │           ├── current
+//	        │           │   └── link
+//	        │           └── index
+//	        │               └── <algorithm>
+//	        │                   └── <hex digest>
+//	        │                       └── link
+//	        └── _uploads
+//	            └── <id>
+//	                ├── data
+//	                ├── hashstates
+//	                │   └── <algorithm>
+//	                │       └── <offset>
+//	                └── startedat
```

Comment:
Was this the tool? That's seriously impressive!

Reply:
LOL, no this one was some manual elbow-grease 😂 🤷‍♂️ . It did an "ok" job on formatting it (using tabs for indentation), but I still found it hard to read. Given that it was describing a "tree(ish)" output, I thought I'd just use `tree` to do just that.

```bash
mkdir -p "<root>/v2/repositories/<name>/_manifests/revisions/<manifest digest path>"
touch "<root>/v2/repositories/<name>/_manifests/revisions/<manifest digest path>/link"
mkdir -p "<root>/v2/repositories/<name>/_manifests/tags/<tag>/current"
touch "<root>/v2/repositories/<name>/_manifests/tags/<tag>/current/link"
mkdir -p "<root>/v2/repositories/<name>/_manifests/tags/<tag>/index/<algorithm>/<hex digest>"
touch "<root>/v2/repositories/<name>/_manifests/tags/<tag>/index/<algorithm>/<hex digest>/link"
mkdir -p "<root>/v2/repositories/<name>/_layers/<layer links to blob store>"
mkdir -p "<root>/v2/repositories/<name>/_uploads/<id>/hashstates/<algorithm>"
touch "<root>/v2/repositories/<name>/_uploads/<id>/hashstates/<algorithm>/<offset>"
touch "<root>/v2/repositories/<name>/_uploads/<id>/data"
touch "<root>/v2/repositories/<name>/_uploads/<id>/startedat"

tree
```